src/api/routes/categoryRoute.js

```diff
@@ -0,0 +1,8 @@
+import { Router } from 'express';
+import categoryController from '../controllers/categoryController';
+
+const categoryRouter = new Router();
+
+categoryRouter.get('/category', categoryController.getCategory);
```

Comment:
then this would be something like `/all`

Reply:
Why not just '/' here @alainburindi